.github/workflows/test.yml

```
@@ -0,0 +1,140 @@
+
+      - name: Set up Python
+        run: uv python install ${{ matrix.python-version }}
+
+      - name: Install dependencies
+        env:
+          UV_PYTHON: ${{ matrix.python-version }}
+        run: uv sync --all-extras
+
+      - name: Detect tests
+        id: detect-tests
+        run: |
```

actionlint (1.7.10)

[error] shellcheck reported issue in this script: SC2086:info:1:31: Double quote to prevent globbing and word splitting

(shellcheck)